src/Stratis.Bitcoin.Features.PoA/CollateralPoAConsensusFactory.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Text;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities.JsonConverters;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class CollateralPoAConsensusFactory : PoAConsensusFactory
+    {
+        public override IFederationMember DeserializeFederationMember(byte[] serializedBytes)
+        {
+            string json = Encoding.ASCII.GetString(serializedBytes);
+
+            CollateralFederationMemberModel model = Serializer.ToObject<CollateralFederationMemberModel>(json);
```

Comment:
Also, what if `model` ends up being null?

Reply:
We will throw on the next line.